src/App.tsx

```diff
@@ -36,187 +82,127 @@ export const App: React.FC = () => {
-            <label className="todo__status-label">
-              <input
-                data-cy="TodoStatus"
-                type="checkbox"
-                className="todo__status"
-                defaultChecked
-              />
-            </label>
-
-            <span data-cy="TodoTitle" className="todo__title">HTML</span>
-            <button
-              type="button"
-              className="todo__remove"
-              data-cy="TodoDeleteButton"
-            >
-              ×
-            </button>
-
-            <div data-cy="TodoLoader" className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-
-          <div data-cy="Todo" className="todo">
-            <label className="todo__status-label">
-              <input
-                data-cy="TodoStatus"
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
-
-            <span data-cy="TodoTitle" className="todo__title">CSS</span>
+        {todosAreLoaded && (
+          <section className="todoapp__main" data-cy="TodoList">
+            {
+              filteredTodos.map(todo => {
+                return (
+                  <div
+                    data-cy="Todo"
+                    className={cn('todo',
+                      { completed: todo.completed })}
+                    key={todo.id}
+                  >
+                    <label className="todo__status-label">
+                      <input
+                        data-cy="TodoStatus"
+                        type="checkbox"
+                        className="todo__status"
+                        defaultChecked
+                      />
+                    </label>
+
+                    <span data-cy="TodoTitle" className="todo__title">
+                      {todo.title}
+                    </span>
+                    <button
+                      type="button"
+                      className="todo__remove"
+                      data-cy="TodoDeleteButton"
+                    >
+                      ×
+                    </button>
+
+                    <div data-cy="TodoLoader" className="modal overlay">
+                      <div
+                        className="modal-background has-background-white-ter"
+                      />
+                      <div className="loader" />
+                    </div>
+                  </div>
+                );
+              })
+            }
+          </section>
+        )}
+        {todosAreLoaded && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="todosCounter">
+              4 items left
+            </span>
+
+            <nav className="filter" data-cy="Filter">
+              <a
+                data-cy="FilterLinkAll"
+                href="#/"
```

Comment:
move it to the separate component, for navigation you also can create a separate component